code/__DEFINES/~sunset_defines/DNA.dm

```diff
@@ -0,0 +1,8 @@
+#define REVIVESBYHEALING 31 // Will revive on heal when healing and total HP > 0.

```

Comment:
You may wish to rename all of these - `TRAIT_`, just for the sake of consistency. These were made before the refactor this codebase now operates under.